addons/vagrant/setup-dev-env.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
I think this would be better in the dev-helpers so that we can use this for other things than for vagrant